k8s/README.md

```diff
@@ -34,7 +34,7 @@ The production branch maintains an image that can be released in production with
   pip install -r ./requirements.txt
   ```
 
-- Create symbolic links to `kubectl` and `kubeconfig` to use in each region
+- If you are deploying to a Kops cluster, create symbolic links to `kubectl` and `kubeconfig` to use in each region
```

Comment:
Can we maybe create two sections similar to what you have to the EKS cluster?

Something like:
`Accessing a kops cluster`
the existing content

`Accessing an EKS cluster`
new content

Probably this will require some duplication of the available information but it will be probably easier to understand and to maintain

Reply:
ok